functions/src/repository/pull-request.ts

```diff
@@ -0,0 +1,68 @@
+import * as firebase from 'firebase-admin';
+import { FirebaseAdmin, QuerySnapshot } from '../client/firebase-admin';
+import { GitHubClient } from '../client/github';
+import { Logger } from '../client/logger';
+
+import { BuildTimes, GitHubPullRequestStatusInput, GitHubPullRequestStatusMapper, GitHubPullRequestStatusModel } from '../mappers/github/status.mapper';
+
+/**
+ * Return the build times for all type of context for any PR
+ * @param statuses GitHubPullRequestStatusModel[]
+ */
+export const getPRBuildTime: any = (statuses: GitHubPullRequestStatusModel[]): BuildTimes[] => {
```

Comment:
I think the function name looks a bit weird with `PR` in the middle, might be best to use the full name `PullRequest`?

Reply:
It will then match https://github.com/DashboardHub/PipelineDashboard/pull/1577/files#diff-506ec91408439735e448bd31cbc18576R31

Reply:
okay